Dockerfile

```
@@ -0,0 +1,17 @@
+# 第一阶段：使用 Python 镜像安装 Python 和需要的 Python 包
+FROM python:slim AS python-base
+RUN pip install uv
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)